docs/opencompetition.md

```diff
@@ -0,0 +1,116 @@
+# Open Optimization Competition 2020
+
+The Nevergrad and IOHprofiler teams are happy to announce that we have 
+paired up for the Open Optimization Competition 2020. 
+
+
+
+## Overview 
+All kind of contributions (under the form of pull requests) to Nevergrad an IOHProfiler are eligible.
+For example, we distinguish two (non exclusive) categories for submission.
+
+### Performance: Improvements for any of the following performance tracks
+(classic competition, with existing baselines that are meant to be outperformed)
+- One-shot optimization
+- Low budget optimization
+- Multi-objective optimization
+- Discrete optimization
+- Structured optimization 
+- Constrained optimization
+We explicitly allow portfolio-based algorithms such as landscape-aware algorithm selection techniques, etc. 
+
+### New ideas & others: contributions to the experimental routines, including 
+- suggestions for performance measures and statistics, 
+- good benchmark problems, 
+- visualization of data, 
+- parallelization, 
+- reproducibility, etc. 
+
+## Submission: 
+All contributions are directly made to either one of the tools, Nevergrad or IOHprofiler, via pull requests on our GitHub pages. You do not have to do anything else than a pull request.
+https://github.com/facebookresearch/nevergrad for Nevergrad 
+https://github.com/IOHprofiler/ for IOHprofiler [for IOHprofiler, please use the competition branch for your submissions]
+All supporting material should be uploaded together with the pull request. Links to arXiv papers etc are possible and welcome, but by no means mandatory. Keep in mind that a good description of your contribution increases the chance that jury members will understand and value your contribution.  
+All pull requests not yet merged on December 1 2019 or opened before June 1, 2020 are eligible for  the competition
+
+## Key principles:
+- Open source: no matter how good are the results, if they can not be reproduced or the code can not be checked this is not in the scope.
+- Reproducibility: if the code can not be run easily, it is also not in the scope.
+
+## Dates: 
+		Submission deadline is June 1, 2020, AoE
```

Comment:
This does not render very well, maybe use bold instead? (using 2 stars "**" at each end of the text)

Reply:
Done.